synapse/tests/test_cortex.py

```diff
@@ -2210,115 +2174,6 @@ def test_cortex_rev0_savefd_lmdb(self):
             with s_cortex.openurl(lmdb_url, savefile=savefp) as core:
                 self.runrev0savefile(core)
 
-    def test_cortex_rev0(self):
```

Comment:
is there a reason behind removing all the rev0 db things? if so there are still additional items which are not cleaned up from doing so.